confidential/confidential_test.go

```diff
@@ -413,3 +413,18 @@ func TestSurjectionProof(t *testing.T) {
 	}
 
 }
+
+func TestSatoshiToElementsValueAndBack(t *testing.T) {
+	var satoshi uint64 = 200000000
```

Comment:
can we use a random number generator function?

Reply:
Done